py/configu/core/config_schema.py

```diff
@@ -86,6 +92,13 @@ def __init__(self, path: str) -> None:
         super().__init__(path=path, type=ConfigSchema.TYPES[Path(path).suffix])
 
     def read(self) -> str:
+        """
+        Reads the config schema file
+        :return: contents of the config schema
+        """
+        import pdb
+
+        pdb.set_trace()
```

Comment:
@jamesL92 is it a here by mistake?

Reply:
it is indeed :man_facepalming: - removed.